scipy/optimize/_shgo.py

```diff
@@ -766,50 +817,61 @@ def find_minima(self):
         else:
             self.find_lowest_vertex()
 
+        if self.disp:
+            logging.info(
+                "Minimiser pool = SHGO.X_min = {}".format(self.X_min))
+            logging.info("Minimiser pool = SHGO.X_min = {}".format(self.X_min))
+
     def find_lowest_vertex(self):
         # Find the lowest objective function value on one of
         # the vertices of the simplicial complex
-        if self.sampling_method == 'simplicial':
-            self.f_lowest = np.inf
-            for x in self.HC.V.cache:
-                if self.HC.V[x].f < self.f_lowest:
-                    self.f_lowest = self.HC.V[x].f
-                    self.x_lowest = self.HC.V[x].x_a
-            if self.f_lowest == np.inf:  # no feasible point
-                self.f_lowest = None
-                self.x_lowest = None
-        else:
-            if self.fn == 0:
-                self.f_lowest = None
-                self.x_lowest = None
-            else:
-                self.f_I = np.argsort(self.F, axis=-1)
-                self.f_lowest = self.F[self.f_I[0]]
-                self.x_lowest = self.C[self.f_I[0]]
+        self.f_lowest = np.inf
+        for x in self.HC.V.cache:
+            if self.HC.V[x].f < self.f_lowest:
+                if self.disp:
+                    logging.info(f'self.HC.V[x].f = {self.HC.V[x].f}')
+                self.f_lowest = self.HC.V[x].f
+                self.x_lowest = self.HC.V[x].x_a
+        for lmc in self.LMC.cache:
+            if self.LMC[lmc].f_min < self.f_lowest:
+                self.f_lowest = self.LMC[lmc].f_min
+                self.x_lowest = self.LMC[lmc].x_l
+
+        if self.f_lowest == np.inf:  # no feasible point
+            self.f_lowest = None
+            self.x_lowest = None
 
     # Stopping criteria functions:
     def finite_iterations(self):
+        mi = min(x for x in [self.iters, self.maxiter] if x is not None)
+        logging.info(f'Iterations done = {self.iters_done} / {mi}')
+        logging.info(f'Iterations done = {self.iters_done} / {mi}')
```

Comment:
Should these be wrapped by a `self.disp:` block?

Reply:
@Stefan-Endres in case you missed this, I think it's still a relevant question.

Reply:
Yeah sorry I missed this @Kai-Striega is right, I will do a quick commit.